www/restinio/files/patch-Doxyfile.diff

```diff
@@ -0,0 +1,4989 @@
+--- dev/Doxyfile.orig	2018-04-30 18:35:55.000000000 -0300
++++ dev/Doxyfile	2018-04-30 18:38:21.000000000 -0300
+@@ -1,2493 +1,2493 @@
```

Comment:
Scratching my head a bit. Is this a 5k line patch file which merely converts CRLF to LF? I didn't check, I'm honestly asking. If so, I would kind of prefer to add a dependency on `dos2unix` and run the tool in post-patch phase. Or run some regexp or something (I'm sure there's a more straightforward way that one that requires an additional dependency). In any case storing a huge diff for the sake of lineendings alone seems like an overkill.